example.py

```diff
@@ -26,25 +26,21 @@ def print_scores(scores):
         for column in scores[interviewer].keys():
             
             score = scores[interviewer][column]['score']
-            observed_frequencies = scores[interviewer][column]['observed_freq']
-            expected_frequencies = scores[interviewer][column]['expected_freq']
-            p_value = scores[interviewer][column]['p_value']
 
-            print("Observed Frequencies: %s" % observed_frequencies)
-            print("Expected Frequencies: %s" % expected_frequencies)
-            print("P-Value: %d" % p_value)
+            print("Question: %s" % column)
+            print("Score: %d" % score)
+
+            # observed_frequencies = scores[interviewer][column]['observed_freq']
+            # expected_frequencies = scores[interviewer][column]['expected_freq']
+            # p_value = scores[interviewer][column]['p_value']
+            # print("Observed Frequencies: %s" % observed_frequencies)
+            # print("Expected Frequencies: %s" % expected_frequencies)
+            # print("P-Value: %d" % p_value)
 
 if __name__ == '__main__':
     data = pd.read_csv(DATA_FILE)  # Uncomment to load as pandas.DataFrame.
-    # data = mlab.csv2rec(DATA_FILE)  # Uncomment to load as numpy.recarray.
-
-    # Compute SVA outlier scores.
-    (sva_scores, _) = outlierdetect.run_sva(data, 'interviewer_id', ['cough', 'fever'])
-    print("SVA outlier scores")
-    print_scores(sva_scores)
-
-    # Compute MMA outlier scores.  Will work only if scipy is installed.
-    if hasattr(outlierdetect, 'run_mma'):
```

Comment:
Let's keep this check in as well since it is possible that we won't have the `run_mma` method if we haven't installed `scikit` (I think)